virtual_ship/location.py

```diff
@@ -7,23 +7,23 @@
 class Location:
```

Comment:
just swapping y,x for x,y. changed the rest of the code accordingly

Reply:
Hmm, while it's often more common to give locations in (lon, lat), there is also a point to be made to keep the (lat, lon) order, as Parcels for examples samples like (time, depth, lat, lon). @ammedd, what do you think?

Reply:
Lat, lon. It is used everywhere with python (and already confusing enough as it is)

Reply:
Ok, ordered it lat lon, y x, V U everywhere